utils/DetailsScoreCalculation.js

```diff
@@ -0,0 +1,47 @@
+/* The following functions help calculate the accordance and deviation 
+of consumption of INDIVIDUAL FOOD CATEGORIES */
+
+// The following function calculates the (percentage) deviation from the recommended consumption (by Planetary Health Diet)
+
+export function DetailsScoreCalculationDeviation(
```

Comment:
Usually, function names start with a verb: `calculateDetailsScoreDeviation`; I suggest following this practice here as well.

Besides: nice work!

Reply:
Implemented that, thanks alot! :-)